src/app/pages/signing/signing.component.css

```
@@ -135,6 +140,47 @@ markdown {
   border-radius: 0.4em;
 }
 
+/* || Embed Photo */
+
+#disclaimer-button {
+  color: white;
+}
+
+#embed-image-holder {
+  max-width: 100%;
+}
+
+#embed-image-card {
+  border: 0.0625em solid grey;
+  min-height: calc(1.5em + .75rem + 2px);
```

```suggestion
  min-height: 2.375em;
```
Yeah lmao I think I copied the height of another component that I saw through the developer console and never converted back to em.